### ISSUE TITLE
How to train this model on celebA dataset

### ISSUE THREAD

**Issue by ljjcoder** [JiahuiYu/generative_inpainting#97]
Hi JiahuiYu，I want to train this model on celebA dataset, but I can not get the results as good as your provided results. I used the default train set on celebA and I trained our model about 5 days on TITAN X. 
![image](https://user-images.githubusercontent.com/40750189/43046329-dd82423c-8df9-11e8-84d4-ef297637bb19.png)
![image](https://user-images.githubusercontent.com/40750189/43046331-e1c23852-8df9-11e8-8ac7-d04a0b71904e.png)
I set the "random_crop" as True like this
![image](https://user-images.githubusercontent.com/40750189/43046370-584844c6-8dfa-11e8-999a-24590466d840.png)
I changed the path of celebA dataset as below
![image](https://user-images.githubusercontent.com/40750189/43046371-5d9c3a9a-8dfa-11e8-83a9-367611a39c0c.png)
The content of the celebA_trian.txt is showed below
![image](https://user-images.githubusercontent.com/40750189/43046372-5f68300e-8dfa-11e8-83c6-b836fa85ba3b.png)
Is there anything I miss?

**Comment by JiahuiYu** [comment]
@ljjcoder Please shuffle the file list in celebA_train.txt to make it random to some extent. I think that's one difference. Other hyper-parameters are correct as your screen shot.

**Comment by ljjcoder** [comment]
Can you give me your shuffled file list?

**Comment by JiahuiYu** [comment]
@ljjcoder You can use linux command `shuf celebA_train.flist > celebA_train_shuffled.flist` to generate file list. Please have a reference of [this file list](https://drive.google.com/drive/folders/1lpluFXyWDxTY6wcjixQGWX8jxUUMlyBW).

**Comment by Gavin666Github** [comment]
@ljjcoder Hi,brother，my shuffled file list is like this，You can refer to it

![image](https://user-images.githubusercontent.com/35061171/43190543-e3e19ec8-902b-11e8-88ae-6aa3403248c2.png)

**Comment by ljjcoder** [comment]
@Gavin666Github@JiahuiYu  Thanks for your help。I will try it.

**Comment by ljjcoder** [comment]
@JiahuiYu hi,JiahuiYu,I shuffed the celebA_train.flist, and retrain the model for five days, but the results are still not very good. The following is the test result and the convergence curve of loss in the training process。Do you have some advice to get  better result? Thank you very much.
 
![13000_celebahr_patches_163050_input](https://user-images.githubusercontent.com/40750189/43363802-74519d62-933f-11e8-9bf3-874ba2595438.png)
![13000_celebahr_patches_164787_input](https://user-images.githubusercontent.com/40750189/43363804-77d46e60-933f-11e8-9833-35820c8e388a.png)
![13000_celebahr_patches_165118_input](https://user-images.githubusercontent.com/40750189/43363805-79a7fe50-933f-11e8-915a-c4d01705adda.png)
![13000_celebahr_patches_165230_input](https://user-images.githubusercontent.com/40750189/43363807-7c84a830-933f-11e8-858d-cd7bf819eb90.png)

![6e88cf6969bb96e1a37c25d2a673353](https://user-images.githubusercontent.com/40750189/43364255-57627f6e-9349-11e8-8367-9e87d0679144.png)
![6446a14a8df3e5ccbaa933d3c15b42e](https://user-images.githubusercontent.com/40750189/43364257-698ea032-9349-11e8-8ada-6097c5e59741.png)

![5f172c5fa94c58a2d1697fe91213709](https://user-images.githubusercontent.com/40750189/43364264-8953a7d2-9349-11e8-8112-26a480306923.png)
The

**Comment by JiahuiYu** [comment]
@ljjcoder I guess you will need to train more epochs since your g_loss is not even converging. You could have a look of these curves in issue #86 and compare tag gan/global_gan/g_loss.

As a reference, I trained 5 days on 1080Ti GPU with GPU utilization around 99%.
